### ISSUE TITLE
Trace text background assumed to be always dark

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#8055]
When looking at My Traces / edit, what is not too hard to read on a dark background:
![13731-3](https://user-images.githubusercontent.com/1230959/95146557-b9562080-07b0-11eb-9bd4-f66bcce1d197.jpg)
Becomes a usability disaster on a light background:
![24736-2](https://user-images.githubusercontent.com/1230959/95146570-c410b580-07b0-11eb-9c6c-4380c76bd118.jpg)
I wonder if ditching the magenta for plain white on black would fix it for all cases.

**Comment by quincylvania** [comment]
Yeah, I never loved the pink. There are lots of ways we could make custom data nicer, but this is an easy one.
